src/content/cloud/credentials.mdx

```diff
@@ -89,7 +89,9 @@ No resources found.
 
 ## Using your Kubernetes credentials
```

Comment:
should we add info about use `OKTETO_USE_STATIC_KUBETOKEN` flag?

Reply:
`OKTETO_USE_STATIC_KUBETOKEN` is a temporal feature flag that allows the user to use the old auth method using static tokens

Reply:
Already added documentation for the flag. I'll resolve the comment